perfdash/metrics-downloader.go

```diff
@@ -34,39 +34,34 @@ import (
 	"k8s.io/kubernetes/test/e2e/perftype"
 )
 
-// GoogleGCSDownloaderOptions is an options for GoogleGCSDownloader.
-type GoogleGCSDownloaderOptions struct {
+// DownloaderOptions is an options for Downloader.
+type DownloaderOptions struct {
 	ConfigPaths        []string
 	GithubConfigDirs   []string
 	DefaultBuildsCount int
 	LogsBucket         string
 	LogsPath           string
-	CredentialPath     string
 	// Development-only flag.
 	// Overrides build count from "perfDashBuildsCount" label with DefaultBuildsCount.
 	OverrideBuildCount bool
 }
 
-// GoogleGCSDownloader that gets data about Google results from the GCS repository.
-type GoogleGCSDownloader struct {
-	GoogleGCSBucketUtils *bucketUtil
-	Options              *GoogleGCSDownloaderOptions
+// Downloader that gets data about results from a storage service (GCS) repository.
+type Downloader struct {
+	MetricsBuckets MetricsBucket
```

Comment:
MetricsBuckets -> MetricsBucket ?

Reply:
Changed to MetricsBkt to reflect that we are just working with a singular bucket here (as discussed in our offline convo)